### PR TITLE
SW-4728 Do not show projects filter in inventory table views when there are no projects, refresh projects when the component loads

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2098?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #2096** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2096?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #2097** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2097?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #2098** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2098?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2098?utm_source=stack-comment).

File: src/scenes/InventoryRouter/Search.tsx [hunk 1]
```diff
@@ -84,6 +85,10 @@ export default function Search<T extends { facilityInventories?: string }>(props
     void dispatch(requestSubLocations(filters.facilityIds ?? []));
   }, [filters.facilityIds, dispatch]);
 
+  useEffect(() => {
+    void dispatch(requestProjects(selectedOrganization.id, activeLocale || undefined));
+  }, [filters.facilityIds, dispatch, selectedOrganization.id, activeLocale]);
```
Comment (karthikbtf) [inline src/scenes/InventoryRouter/Search.tsx:90]: should the facilityIds be used in the request?

**Review by karthikbtf — APPROVED** [review]
one comment, otherwise looks good

File: src/scenes/InventoryRouter/Search.tsx (hunk 1 above)
Comment (nickgraz) [reply src/scenes/InventoryRouter/Search.tsx:90]: Nope, good catch.

**Comment by nickgraz** [comment]
### Merge activity

* **Jan 26, 5:35 PM**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2098).

**Pull request merged**